manual/vcredist2008/tools/chocolateyUninstall.ps1

```diff
@@ -0,0 +1,15 @@
+﻿$packageName = 'vcredist2005'
+$packageSearch = "Microsoft Visual C++ 2008 Redistributable*"
```

Comment:
> chars wouldn't make a difference I think, this is not regex.

oh right, mixed it up with the `-match` operator.
I guess we could scratch this down to appveyor having multiple versions of vcredist 2008 installed then perhaps.